src/common/feature-flags.ts

```diff
@@ -69,10 +69,9 @@ export function getAllFeatureFlagDetails(): FeatureFlagDetail[] {
         },
         {
             id: FeatureFlags.newIssueFilingExperience,
-            defaultValue: false,
+            defaultValue: true,
             displayableName: 'New issue filing experience',
-            displayableDescription:
-                'Enable the new issue filing experience. Needs "Issue filing" preview feature to be enable in order to work',
+            displayableDescription: 'Enable the new issue filing experience.',
             isPreviewFeature: false,
             forceDefault: true,
```

Comment:
I think you should change `forceDefault` to `false`, right?

Reply:
Right, since we do not want the user to see the old (and broken) UX.

Reply:
FYI, filing a bug from Target page is broken in the old UX.

Reply:
Force default should have been true before in all honesty; could've made it a bit easier to work on 😄

Reply:
I'll get this in after Fer/Mark/Chelsea reviews the UX.